uportal-war/src/main/java/org/jasig/portal/groups/pags/dao/EntityPersonAttributesGroupStore.java

```diff
@@ -317,7 +317,7 @@ private GroupDefinition initGroupDef(IPersonAttributesGroupDefinition group) {
             TestGroup tg = new TestGroup();
             Set<IPersonAttributesGroupTestDefinition> tests = testGroup.getTests();
             for(IPersonAttributesGroupTestDefinition test : tests) {
-                IPersonTester testerInst = initializeTester(test.getTesterClassName(), test.getAttributeName(), test.getTestValue());
+                IPersonTester testerInst = initializeTester(test);
                 tg.addTest(testerInst);
```

Comment:
> perhaps the portal should not operate at all.

Eeep. Please no.  Shrill logging about something being horribly wrong, sure, by all means.  Causing the whole portal to fail, that's aggressively unfriendly.  Seldom is a production outage preferable to uptime of a gracefully degraded service.